master/getting-started/kubernetes/installation/hosted/kubernetes-datastore/calico-networking/1.7/calico.yaml

```diff
@@ -97,6 +179,11 @@ spec:
             # Use Kubernetes API as the backing datastore.
             - name: DATASTORE_TYPE
               value: "kubernetes"
+            # Enable Typha support, Felix will look up the Typha service using this name
+            # and connect to Typha.  To stop Felix from using Typha, remove this environment
+            # variable.
+            - name: FELIX_TYPHAK8SSERVICENAME
+              value: "calico-typha"
```

Comment:
Wonder if it makes sense to have typha config in a config map, making it easy to opt in.

e.g.

```
# Set >0 to enable. 
typhaReplicas: 0
# Set to 'calico-typha' to enable.
typhaService: ""
```

Reply:
Happy to merge without this, but worth considering if we want to make Typha off-by-default but easy to turn on when running at higher scale.

Reply:
Can you have config maps that control things like replica counts?  Agree, could be a good way to get Typha easy to use without the risk of on-by-default.

Reply:
Er, hm, no I suppose you can't. 

I'm not sure what I was thinking there, but for some reason I felt we could do it. I'll have another think.

hand-wavey: maybe we need a typha controller that manages a typha deployment for you.